src/address/address.cpp

```diff
@@ -14,12 +16,24 @@ void printIP6(std::string ip6_str) {
   std::cout << ip6 << std::endl;
 }
 
+void printSocket(std::string host_str, uint16_t port) {
```

Comment:
Actually, this test should be in a completely different folder and test file: src/socket
The idea is that each test is minimal and only test one single thing. That gives us better signals if something goes wrong